packages/expo-notifications/android/src/main/java/expo/modules/notifications/Utils.kt

```diff
@@ -17,3 +26,77 @@ internal fun createDefaultResultReceiver(
     }
   }
 }
+
+/**
+ * Returns true if object is convertible to a WritableMap element
+ */
+internal fun isWritableMapConvertible(value: Any?): Boolean {
```

Comment:
This seems a bit fragile and likely to cause issues if new types are added that we forgot to handle here, or if support for a new type is added to WritableMap. Is it possible to use something like recursive calls to `Arguments.fromBundle` wrapped with try/catches to know what can be serialized?

Reply:
We could do exactly that, but it would be much less performant. This method is actually more or less copied from our own code in `expo-modules-core`, specifically `JSTypeConverter`. So I can add a note here that any changes in `JSTypeConverter` may require changes here.

Reply:
I've moved the test for convertibility of an object to `JSTypeConverter`, since `expo-notifications` has no visibility into internal RN classes like `WritableMap`. The new method is no longer checking through a list of hardcoded types -- it is calling the same method that causes the crash we are seeing, and catching the exception.